grizli/pipeline/auto_script.py

```diff
@@ -4006,7 +3976,7 @@ def make_combined_mosaics(root, fix_stars=False, mask_spikes=False, skip_single_
         if fill_mosaics == 'grism':
             # Only fill mosaics if grism filters exist
             has_grism = utils.column_string_operation(info['FILTER'],
```

Comment:
See again comment above for `has_grism`.  Maybe just set `info.meta['HAS_GRISM']` once when it's generated to avoid having to test it every time